src/engine/enginemaster.h

```diff
@@ -46,6 +46,8 @@ class EngineSync;
 class EngineTalkoverDucking;
 class EngineDelay;
 
+static const unsigned int kMaxChannels = 32;
```

Comment:
We should probably bump this to 64 since we're pretty close to the limit and users may want to make skins with more than 4 decks.

* 16 sample decks (in some skins)
* 4 decks
* 1 preview deck
* 4 microphone
* 4 aux
= 29